<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>specTagsWindow</class>
 <widget class="QMainWindow" name="specTagsWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>269</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>374</width>
    <height>269</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>374</width>
    <height>269</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>تگ ها  - ویدئو ویژه</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/tags.png</normaloff>../icons/tags.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="mainGroupBox">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>6</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>تنظیم تگ ها -  (هر تگ در یک خط مجزا وارد شود)</string>
    </property>
    <widget class="QPushButton" name="cancelPushButton">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>229</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>انصراف</string>
     </property>
    </widget>
    <widget class="QPushButton" name="savePushButton">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>229</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ذخیره و خروج</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="tagsTextEdit">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>20</y>
       <width>345</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
